libopae/plugins/xfpga/sysfs_int.h

```diff
@@ -35,6 +35,9 @@
 
 #define SYSFS_PATH_MAX 256
 
+#define FPGA_SYSFS_CLASS_PATH_INTERNAL "/sys/class/fpga"
+#define FPGA_SYSFS_CLASS_PATH_UPSDRV  "/sys/class/fpga_region"
```

Comment:
can we rename these to use the suffixes:
_INTEL (instead of INTERNAL)
_DFL (instead of UPSDRV)

Reply:
fixed